spec/controllers/v1/sessions_controller_spec.rb

```diff
@@ -187,10 +200,10 @@ def expect_logger_msg(level, msg)
 
       it 'redirect user to external site' do
         LoginRedirectApplication.create(
-          uuid: login_uuid, redirect_application: 'cerner'
+          uuid: login_uuid, redirect_application: Settings.sso.ssoe_redirects['myvahealth']
         )
         allow(SAML::User).to receive(:new).and_return(saml_user)
-        expect(post(:saml_callback)).to redirect_to(Settings.sso.ssoe_redirects['cerner'])
+        expect(post(:saml_callback)).to redirect_to(Settings.sso.ssoe_redirects['myvahealth'])
```

Comment:
Looks good, I would consider another test case  here capturing the ultimate intended redirect behavior for an unknown redirect_application.

Reply:
this test case captures it from perspective of when authentication starts, https://github.com/department-of-veterans-affairs/vets-api/pull/4055/files#diff-b6ff071088096f7d49f49f8b428ab449R133-R144